weave/tests/test_weave_client.py

```diff
@@ -239,6 +239,65 @@ def test_calls_query(client):
     client.finish_call(call0, None)
 
 
+def test_calls_delete(client):
```

Comment:
please verify that you ran these tests in clickhouse mode

Reply:
@tssweeney im sure this is easy to do but for the life of me I can't figure out the magic incantation...